scientist-softserv/harvard-curiosity
# Add Set spec to indexing table

**Add Set spec to indexing table.**
* * *

**JIRA Ticket**: [(#217)](https://github.com/harvard-lts/CURIOSity/issues/217)

# What does this Pull Request do?
This adds the Set Spec of the Harvest to the "Recent item indexing activity" of the Dashboard.

# How should this be tested?

A description of what steps someone could take to:
* Pull in the code from the `reindexing-table` branch
* Rebuild the application
* Login as an admin
* Go to any exhibit dashboard
* Confirm that the Set Spec column exists in the "Recent item indexing activity" table.
* Harvest an exhibit and confirm the correct Set Spec displays in the "Recent item indexing activity" table of the dashboard.

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? No
- integration tests? No
- 
# Interested parties
@mferrarini @dl-maura
